### PR TITLE
Adding term "tribe" to tier-3

### PR DESCRIPTION
**Pull request by LarryKunz**

### CODE REVIEW

**Review by abuango — APPROVED** [review]
LGTM

**Comment by abuango** [comment]
@LarryKunz Is this ready for merge?

**Comment by quaid** [comment]
> @LarryKunz Is this ready for merge?

I think so, I'll do my review and give approvals. But we discussed on Friday waiting through the weekend so we can point to the activity on Monday (in Slack, etc.) where people can watch the PR in progress. (I know it's the same as a merged PR, but the feeling is different when it hasn't been merged yet.) Since part of doing this is enabling everyone in the work stream to do at least one pull request of one of the words/terms, we thought a few days pause was a good idea.

File: content/language/word-list/tribe.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Tribe
+---
+
+**Definition**
+
+*Tribe* is a term that can be associated with colonialism or can be considered cultural appropriation.
```
Comment (quaid) [inline content/language/word-list/tribe.md:7]: I like this practice of using italics the first time the word appears in the document; this is (I'm sure Larry knows) the common practice in technical materials -- denoting the first usage of a term by italicizing it and also having that place in the document either define the term or make it clear what it means in that context.

**Review by quaid — APPROVED** [review]
/lgtm
/approve

**Pull request merged**